src/store/actions.ts

```diff
@@ -56,25 +56,34 @@ export default {
   setCurrentFilingStatus ({ commit }, currentFilingStatus: FilingStatus) {
     commit('currentFilingStatus', currentFilingStatus)
   },
-  setTasks ({ commit }, tasks: Array<object>) {
+  setTasks ({ commit }, tasks: Array<any>) {
     commit('tasks', tasks)
   },
-  setFilings ({ commit }, filings: Array<object>) {
+  setFilings ({ commit }, filings: Array<LedgerIF>) {
     commit('filings', filings)
   },
-  setRegisteredAddress ({ commit }, registeredAddress: object) {
+  setRegisteredAddress ({ commit }, registeredAddress: OfficeAddressIF) {
     commit('registeredAddress', registeredAddress)
   },
-  setRecordsAddress ({ commit }, recordsAddress: object) {
+  setRecordsAddress ({ commit }, recordsAddress: OfficeAddressIF) {
     commit('recordsAddress', recordsAddress)
   },
-  setDirectors ({ commit }, directors: Array<object>) {
+  setDirectors ({ commit }, directors: Array<DirectorIF>) {
     commit('directors', directors)
   },
-  setLastAnnualReportDate ({ commit }, lastAnnualReportDate: string) {
-    commit('lastAnnualReportDate', lastAnnualReportDate)
+  setLastAnnualReportDate ({ commit }, date: string) {
+    commit('lastAnnualReportDate', date)
   },
-  setConfigObject ({ commit }, configObject: object) {
+  setLastFilingDate ({ commit }, date: Date) {
```

Comment:
This is a good eample of some dates being Dates and others (above) being strings. Just be careful when using them. Sometimes they appear compatible (eg, printing a Date gives you a string) but they generally are not.